python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -59,7 +60,10 @@ def __init__(self, verbose=True, lonCol='fieldRA', latCol='fieldDec',
         self.slicePoints['ra'] = None
         self.slicePoints['dec'] = None
         self.nslice = None
+        self.shape = None
         self.plotFuncs = [BaseHistogram, BaseSkyMap]
+        # Values in self.slicePoints to pass whole to the metrics
+        self.bulkSlicePointKeys = []
```

Comment:
You might move this all the way up to the BaseSlicer so that all kinds of slicers have this attribute (in case we extend 2d slicing stuff to the oneDslicer, which might replace something like the hourglass slicer). 

bulkSlicePointKeys ... should this be replaced by something that clearly indicates that its' the values for the 2d subfunctionality of the slicers? (and I keep getting a little confused by what a 2d slicer means because the spatial slicers are already sort of two-d, at least in their metadata, and there is actually a ND slicer ... it's like they're slicers, with a special subset of 2d metrics that they use, is what I have to keep reminding myself, rather than that the slicer itself is slicing in 2 dimensions).